.github/workflows/github-actions-deploy-snapshots.yml

```
@@ -12,10 +12,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)